### ISSUE TITLE
Feature request: Render select page button next to text input

### ISSUE THREAD

**Issue by thendricks86** [AuraQ/AutoCompleteForMendix#20]
It would be great if it were possible to optionally enable a select button to be rendered next to the control. In some cases, it might be difficult to search in a text box alone as it might be necessary to filter down in order to find an object. I solved this at the moment by putting a standard reference selector next to the auto complete widget, setting it to select by page and disabling everything but the button itself using CSS:

![autocomplete_select1](https://cloud.githubusercontent.com/assets/24310340/20825194/6cf94d04-b89d-11e6-8a0f-e37041945dcf.JPG)

The result looks like this:
![autocomplete_select2](https://cloud.githubusercontent.com/assets/24310340/20825229/b3713cec-b89d-11e6-872c-3b12f3746e1d.JPG)

Clicking the select button will open a select page to set the reference instead. Creating the form like this is totally workable but it would be very cool if the widget had an option to render a select button next to it by default.

**Comment by lindski** [comment]
If you want to pop up a search would it not just be easier to have a standard reference selector on your page? Or do you want both the auto complete functionality plus the option for a more detailed search via a pop up?

**Comment by thendricks86** [comment]
Yes, I am looking for a way to have both options. Example: I might have a CRM application in which I can add contacts to a document. I likely know the name of the person I want to add, so I can just type the name. There are however, also instances where I might only know that I am looking for the assistant IT manager of client A. In that case it would be nice to have a selection grid in a popup in which I can search multiple text fields (like the standard reference selector), without having to display all the data in the search field.

**Comment by lindski** [comment]
I took a look at this over Christmas... there is no way (that I could find) to open a reference selector 'select' page using the client api so all I would be able to do would be to render the button and open a pop up page... the developer would have ensure that they set the correct association from the pop up before returning to the main page.

Does that sound reasonable?

**Comment by pratleymatthew** [comment]
Closed as this can be addressed by standard functionality
